### PR TITLE
improve readability of hard coded names

### PR DESCRIPTION
**Pull request by quleuber**

### CODE REVIEW

File: src/hvm.rs [hunk 1]
```diff
@@ -3623,21 +3623,38 @@ pub fn name_to_u128(name: &str) -> u128 {
...
+  }
+}
+
+pub const fn const_name_to_u128(name: &str) -> u128 {
+  let name = name.as_bytes();
+  let last = name.len() as isize - 1;
+  let mut res: u128 = 0;
+  macro_rules! set {
+    ( $I:expr ) => {
+      let i = $I; let pos = last-i; if pos >= 0 { res = res | (char_to_u128(name[pos as usize] as char) << (i*6)); }
```
Comment (VictorTaelin) [inline src/hvm.rs:3648]: This is much less readable. I don't think the benefit of the consts is worth making this function less clear.

File: src/node.rs [hunk 2]
```diff
@@ -919,9 +919,9 @@ pub fn node_loop(
       }
 
       // Receives and handles incoming network messages
-      if tick % 1 == 0 {
-        node_receive_message(&mut node);
-      }
+      // if tick % 1 == 0 {
```
Comment (VictorTaelin) [inline src/node.rs:922]: I think it is better to leave this if. Yes it is redundant, but it is about following the same structure as the others.

File: src/hvm.rs (hunk 1 above)
Comment (quleuber) [reply src/hvm.rs]: I just did some refactoring, but i'm not satisfied with it yet.

Comment (quleuber) [reply src/hvm.rs]: Btw, the non const function still exists.

**Pull request closed without merging**